store/mongo/datastore_mongo.go

```diff
@@ -413,17 +426,17 @@ func (db *DataStoreMongo) GetAllJobs(
 	sortField := bson.M{}
 	sortField["insert_time"] = -1
 	findOptions.SetSort(sortField)
-	cur, err := collection.Find(ctx, bson.M{}, findOptions)
+	cur, _ := collection.Find(ctx, bson.M{}, findOptions)
 
 	var jobs []model.Job
-	err = cur.All(ctx, &jobs)
+	err := cur.All(ctx, &jobs)
 	if err == mongo.ErrNoDocuments {
 		return []model.Job{}, 0, nil
 	} else if err != nil {
 		return []model.Job{}, 0, err
 	}
 
-	count, err := collection.CountDocuments(ctx, bson.M{})
+	count, _ := collection.CountDocuments(ctx, bson.M{})
```

Comment:
We should return the error, not swallowing it:
```suggestion
      count, err := collection.CountDocuments(ctx, bson.M{})
      if err != nil {
              return []model.Job{}, 0, err
      }
```